app_test_seqan_tcoffee fails on ppc64le

```
$ make test
[…]
        Start 379: build_dox
379/434 Test #379: build_dox .........................................................................***Failed    0.15 sec
[…]
421/434 Test #421: app_test_seqan_tcoffee ............................................................***Failed    1.52 sec
        Start 422: ctd_test_seqan_tcoffee
[…]
99% tests passed, 2 tests failed out of 434

Total Test time (real) = 342.71 sec

The following tests FAILED:
        379 - build_dox (Failed)
        421 - app_test_seqan_tcoffee (Failed)
Errors while running CTest
make: *** [Makefile:109: test] Error 8
```

It still does.

```
$ git log -1
commit 4b5169376808dbc688a3fadd754bd0c0e437af55 (HEAD -> master, origin/master, origin/HEAD)
Author: rrahn <rene.rahn@fu-berlin.de>
Date:   Fri Nov 30 11:40:49 2018 +0100

    [FIX] Use default construction to initilaize data members.
$ make test
[…]
99% tests passed, 3 tests failed out of 434

Total Test time (real) = 391.11 sec

The following tests FAILED:
	264 - test_demo_tutorial_journaled_set_solution_online_search_finder (Failed)
	379 - build_dox (Failed)
	421 - app_test_seqan_tcoffee (Failed)
Errors while running CTest
make: *** [Makefile:109: test] Error 8
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ matrix:
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports']
-          packages: ['g++-4.9', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip']
+          packages: ['g++-4.9', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip', 'libxml2-utils']
       install: export CXX="g++-4.9" CC="gcc-4.9"
 
     - os: linux
@@ -19,7 +19,7 @@ matrix:
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports']
-          packages: ['g++-5', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip']
+          packages: ['g++-5', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip', 'libxml2-utils']
       install: export CXX="g++-5" CC="gcc-5"
 
     - os: linux
@@ -27,7 +27,7 @@ matrix:
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports']
-          packages: ['g++-7', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip']
+          packages: ['g++-7', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip', 'libxml2-utils']
       install: export CXX="g++-7" CC="gcc-7"
 
     - os: linux
@@ -35,7 +35,7 @@ matrix:
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports' , 'llvm-toolchain-precise-3.5']
-          packages: ['clang-3.5', 'g++-4.9', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip'] # g++ required for newer libstdc++
+          packages: ['clang-3.5', 'g++-4.9', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip', 'libxml2-utils'] # g++ required for newer libstdc++
       install: export CXX="clang++-3.5" CC="clang-3.5"
 
     # - os: linux
@@ -52,7 +52,7 @@ matrix:
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports' , 'llvm-toolchain-precise-3.9']
-          packages: ['clang-3.9', 'g++-7', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip'] # g++ required for newer libstdc++
+          packages: ['clang-3.9', 'g++-7', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip', 'libxml2-utils'] # g++ required for newer libstdc++
       install: export CXX="clang++-3.9" CC="clang-3.9"
 
 # currently too slow on osx

--- a/include/seqan/graph_align/graph_impl_align_adapt.h
+++ b/include/seqan/graph_align/graph_impl_align_adapt.h
@@ -66,9 +66,7 @@ public:
     TVertexDescriptor_ data_source;
     TEdgeDescriptor_ data_edge;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph_ const& _graph, TVertexDescriptor_ const v) :
         data_host(&_graph),

--- a/include/seqan/graph_types/graph_iterator_outedge.h
+++ b/include/seqan/graph_types/graph_iterator_outedge.h
@@ -85,9 +85,7 @@ public:
     TVertexDescriptor_ data_source;
     TEdgeDescriptor_ data_edge;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph_ const& _graph, TVertexDescriptor_ const v) :
         data_host(&_graph),
@@ -151,9 +149,7 @@ public:
     TVertexDescriptor data_source;
     TEdgeDescriptor data_edge;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph const& _graph, TVertexDescriptor const v) :
         data_host(&_graph),
@@ -217,9 +213,7 @@ public:
     TVertexDescriptor data_source;
     TEdgeDescriptor data_edge;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph const& _graph, TVertexDescriptor const v) :
         data_host(&_graph),
@@ -287,9 +281,7 @@ public:
     TSize_ data_begin;
     TSize_ data_end;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph_ const& _graph, TVertexDescriptor_ const v) :
         data_host(&_graph),
@@ -367,9 +359,7 @@ public:
     TVertexDescriptor_ data_source;
     TEdgeDescriptor_ data_edge;
 
-    Iter()
-    {
-    }
+    Iter() = default;
 
     Iter(TGraph_ const& _graph, TVertexDescriptor_ const v) :
         data_host(&_graph),

